site/pages/create-consequence-operator.page.tsx

```diff
@@ -0,0 +1,222 @@
+import Link from "next/link";
+import { ReactElement, useState } from "react";
+import Radios from "../components/form/Radios";
+import Select from "../components/form/Select";
+import Table from "../components/form/Table";
+import TextInput from "../components/form/TextInput";
+import TimeSelector from "../components/form/TimeSelector";
+import { BaseLayout } from "../components/layout/Layout";
+import { DISRUPTION_SEVERITIES, OPERATORS } from "../constants";
+import { ErrorInfo } from "../interfaces";
+
+const title = "Create Consequence Operator";
+const description = "Create Consequence Operator page for the Create Transport Disruptions Service";
+
+interface CreateConsequenceOperatorProps {
+    inputs: PageState;
+    previousConsequenceInformation: { modeOfTransport: string; consequenceType: string };
+}
+
+export interface ConsequenceOperatorPageInputs {
+    "consequence-operator": string;
+    description: string;
+    "remove-from-journey-planners": string;
+    "disruption-delay": string;
+    "disruption-severity": string;
+    "disruption-direction": string;
+}
+
+export interface PageState {
```

Comment:
Unsure if this interface needs exporting - just because we have one with the same name on the disruptions page. If we can get away with not exporting *either* of them, then lets do that.

If that causes issues, probably need to rename them both, the same as what we did for the props.

Reply:
done